app/routes/wallet/jws.py

```diff
@@ -64,15 +85,31 @@ async def sign_jws(
     "/verify",
     response_model=JWSVerifyResponse,
     summary="Verify JWS",
-    description="""
-Verify JSON Web Signature (JWS)
-
-See https://www.rfc-editor.org/rfc/rfc7515.html for the JWS spec.""",
 )
 async def verify_jws(
     body: JWSVerifyRequest,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> JWSVerifyResponse:
+    """
+    Verify JSON Web Signature (JWS)
+    ---
+
+    See https://www.rfc-editor.org/rfc/rfc7515.html for the JWS spec.
```

Comment:
This still needs a description of what the endpoint does. "This endpoint takes a JWS string in compact serialization format and verifies its authenticity and integrity." (GPT)